Minor changes (#45)
ehn-digital-green-development/hcert-spec: hcert_spec.md

This corresponds to the COSE algorithm parameter `ES256`.

- Secondary Algorithm: The secondary algorithm is RSASSA-PSS as defined in ([RFC 8230](https://tools.ietf.org/html/rfc8230)) with a modulus of 2048 bits in combination with the SHA–256 hash algorithm as defined in (ISO/IEC 10118–3:2004) function 4.

This corresponds to the COSE algorithm parameter: `PS256`.

#### Key Identifier

The Key Identifier (`kid`) claim is used by Verifiers for selecting the correct public key from a list of keys pertaining to the Issuer (`iss`) Claim. Several keys may be used in parallel by an Issuer for administrative reasons and when performing key rollovers. The Key Identifier is not a security-critical field. For this reason, it MAY also be placed in an unprotected header if required. Verifiers MUST accept both options.

Due to the shortening of the identifier (for space-preserving reasons) there is a slim but non-finite chance that the overall list of DSCs accepted by a validator may contain DSCs with duplicate `kid`s. For this reason a verifier MUST check all DSCs with that `kid`.

####  Issuer

> The Issuer (`iss`) claim is a string value thAT MAY optionally hold the ISO 3166-1 alpha-2 Country Code of the entity issuing the health certificate. This claim can be used by a Verifier to identify which set of DSCs to use for validation. The Claim Key 1 is used to identify this claim.

Reviewer: ```suggestion
The Issuer (`iss`) claim is a string value that MAY optionally hold the ISO 3166-1 alpha-2 Country Code of the entity issuing the health certificate. This claim can be used by a Verifier to identify which set of DSCs to use for validation. The Claim Key 1 is used to identify this claim.
```

Such Member State-specific lists are expected to be adapted in the format for their own national setting. As such, the file format of this trusted list may vary, e.g., it can be a signed JWKS ([JWK set format per RFC 7517 section 5](https://tools.ietf.org/html/rfc7517#section-5)) or any other format specific to the technology used in that Member State.

For the sake of simplicity: Member States may both submit their existing CSCA certificates from their ICAO eMRTD systems or, as recommended by the WHO, create one specifically for this health domain. 

## The Key Identifier (`kid`s)

The key identifier (`kid`) is calculated when constructing the list of trusted public keys from DSC certificates and consists of a truncated (first 8 bytes) SHA-256 fingerprint of the DSC encoded in DER (raw) format.

Note that Verifiers do not need to calculate the `kid` based on the DSC certificate and can directly match the key identifier in issued health certificate with the `kid` on the trusted list.

> ## Differences with the ICAO MasterList System for Passports

Reviewer: ```suggestion
## Differences to the ICAO MasterList System for Passports
```

## Secretariat

In order to alleviate the burden of countries during the initial phase, there shall be a secretarial service which will:

### version 1.00 - secretariat tasks

In the first version, the secretariat will:

* Maintain a non-public list of operational and legal contacts for each Member State to further the orderly management of this health specific set of master lists.
* Maintain a public 24x7 incident/security contact point.
* Maintain a public, integrity(secure) protected, single, up to date, aggregated, list of all CSCAs (DGCG).
* Shall validate the DSCs against the CSCA prior to publication (DGCG).
* Maintain a public, integrity(secure) protected,  single, up to date, aaggregated, list of all DSAs thus validated (DGCG).
* Provide Member States with a secure (i.e. integrity protected) mechanism by which the Secretariat publishes the Member States aggregated CSCA and DSC lists (CIRCABC, DGCG, t.b.c)

**In all cases, the secretariat acts not as content owner, all signatures and certificates must be provided by attendees.**

> Note 1: The curent DGCG design allows for list to be up to date in real time (i.e., it is dynamically generated from a database with the most up to date information then available. So the here aggregated is the output of a data query to given parameters against the uploaded data by the member states) The technical requirement is lighter - the lists should be updated within 24 hours of any change submitted by a member state.

Reviewer: ```suggestion
Note 1: The current DGCG design allows for the list to be up to date in real-time (i.e., it is dynamically generated from a database with the most up to date information available at this point in time. So the here aggregated list is the output of a data query to given parameters against the uploaded data by the Member States). The technical requirement is lighter - the lists should be updated within 24 hours of any change submitted by a Member State.
```
